websites/visualization/src/app/react/experimental/await/page.tsx

```diff
@@ -0,0 +1,48 @@
+'use client'
+
+import { ErrorBoundary, Suspense } from '@suspensive/react'
+import { Await } from '@suspensive/react/experimental'
+
+const delay = (ms: number) =>
+  new Promise((resolve) =>
+    setTimeout(() => {
+      resolve('resolved')
+    }, ms)
+  )
+
+const asyncFn = () => delay(4000 * Math.random()).then(() => 'success' as const)
+
+export default function Page() {
+  return (
+    <ErrorBoundary fallback={() => <>error</>}>
+      <Suspense fallback={<>loading...</>}>
+        <Await options={{ key: [2000] as const, fn: asyncFn }}>
+          {(awaited) => (
+            <div>
+              <div>{awaited.data}</div>
+              <div>{awaited.data}</div>
+            </div>
+          )}
+        </Await>
```

Comment:
The reason Cache was removed from the name is that `useQuery` of @tanstack/react-query also uses Cache internally, but uses the more important name `Query` to express querying not `Cache`. Therefore, SuspenseCache was also renamed to focus on `awaiting promise` with the corresponding fn `Await`, not `Cache`.